catalog/spatial/wfs/spatial-wfs-featuretransformer-xstream/src/main/java/org/codice/ddf/spatial/ogc/wfs/transformer/xstream/XStreamWfs20FeatureTransformer.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+
+package org.codice.ddf.spatial.ogc.wfs.transformer.xstream;
+
+import ddf.catalog.data.MetacardType;
+import java.util.Optional;
+import javax.xml.namespace.QName;
+import net.opengis.wfs.v_2_0_0.FeatureTypeType;
+import org.apache.commons.lang3.StringUtils;
+import org.codice.ddf.spatial.ogc.wfs.catalog.converter.FeatureConverter;
+import org.codice.ddf.spatial.ogc.wfs.catalog.converter.FeatureConverterFactory;
+import org.codice.ddf.spatial.ogc.wfs.catalog.mapper.MetacardMapper;
+import org.codice.ddf.spatial.ogc.wfs.v2_0_0.catalog.converter.impl.GenericFeatureConverterWfs20;
+
+public class XStreamWfs20FeatureTransformer extends XStreamWfsFeatureTransformer<FeatureTypeType> {
```

Comment:
This class looks like it does the same thing as `XStreamWfs10FeatureTransformer` and `XStreamWfs11FeatureTransformer`. Do these need to be separate classes?